c7n/resources/ml.py

```diff
@@ -22,6 +22,15 @@ class resource_type(TypeInfo):
         arn_type = "mlmodel"
         permissions_enum = ('machinelearning:DescribeMLModels',)
 
+    def resources(self, query=None, augment=True):
+        try:
+            return super().resources(query, augment)
+        except ClientError as e:
+            # ml not available to new accounts, use sagemaker.
+            if 'no longer available' in str(e):
```

Comment:
would logging a warning here be useful? returning an empty list might be confusing if the caller was expecting results